notebooks/working/investigating_deconfounder.py

```diff
@@ -79,6 +78,7 @@
 # -
 
 # ## Specify the assumed causal graph of the data generation process
+# The causal graph below represents the true data generating process that we'll be using for simulation. Notice how the confounder affects variables a, b, d, e, and f, as well as the outcome y. 
 
 # +
 data_generation_graph = CausalGraphicalModel(
```

Comment:
Might not be necessary, but I would just move the nodes and edges lists to be separate parameters at the top of the notebook.